packages/core/src/cloud/LlamaCloudRetriever.ts

```diff
@@ -0,0 +1,85 @@
+import { PlatformApi, PlatformApiClient } from "@llamaindex/cloud";
```

Comment:
NOTE: This import won't translate to a `require` call (it's just used for type checking by Typescript)